lang/en/auth_saml2.php

```diff
@@ -183,6 +184,9 @@
 $string['test_auth_button_login'] = 'IdP Login';
 $string['test_auth_button_logout'] = 'IdP Logout';
 $string['test_auth_str'] = 'Test isAuthenticated and login';
+$string['test_endpoint'] = 'Connection test URL';
+$string['test_endpoint_desc'] = 'Enter a URL to test connection against for IdP redirection. Some users or networks may not have connectivity to the IdP based on account or network permissions.';
```

Comment:
maybe tweak 2 strings these so its clear it is client network access not server

Reply:
updated